### ISSUE TITLE
Ban "headless" tables

### ISSUE THREAD

**Issue by janajaeger** [SUSE/doc-styleguide#312]
Another accessibility issue coming your way:

In order to present info in tables to a visually impaired reader, screenreaders need context to all cells of the tables. This is provided by a table header entry of each row.

Currently, our docs feature various tables lacking these header entries either partly or entirely. Here is an example for a badly built table from our docs:

![Image](https://github.com/user-attachments/assets/2634bf50-1af4-46f5-8def-2bbcfeb4fe42)

We need to:
- Kill all occurrences of headless tables
- Add some guidance to the accessibility section of the styleguide on why these tables are bad and what alternatives there are. A nice description on why the missing context is bad can again be found in Siteimprove

Once this has been discussed on the tools meeting, I'll pop some more info in here.

**Comment by dariavladykina** [comment]
Implemented, closing.
